tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum                 | 8 per mesh. Can be       | 8 per mesh, 256 per view.| Unlimited.               |
+| OmniLights              | increased.               |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum                 | 8 per mesh. Can be       | 8 per mesh, 256 per view.| Unlimited.               |
+| SpotLights              | increased.               |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum                 | 8                        | 8                        | 8                        |
+| DirectionalLights       |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Maximum clustered       | N/A                      | N/A                      | 512. Can be increased.   |
+| elements                |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| PCSS for                | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| OmniLight and SpotLight |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| PCSS for                | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+| DirectionalLight        |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Light projector         | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| textures                |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+
+Global Illumination
+~~~~~~~~~~~~~~~~~~~
+
+See :ref:`doc_introduction_to_global_illumination` for more information.
+
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Feature                 | Compatibility            | Mobile                   | Forward+                 |
++=========================+==========================+==========================+==========================+
+| ReflectionProbe         | ⚠️ Planned.              | ✔️ Supported.            | ✔️ Supported.            |
```

Comment:
Limits vary depending on renderer. Also, Compatibility supports reflection probes since 4.3:

```suggestion
| ReflectionProbe         | ⚠️ Supported, 2 per mesh.              | ✔️ Supported, 8 per mesh.            | ✔️ Supported, unlimited.            |
```